docs/pages/Troubleshooting.md

```diff
@@ -23,3 +23,22 @@ rm -rf ~/.ui5/framework/
 ```
 
 Any missing framework dependencies will be downloaded again during the next UI5 Tooling invocation.
+
+## UI5 CLI
+### Changing the Log Level via Environment Variable
+
+In CI environments or in a combination with other tools, the usage of [UI5 CLI's `--log-level`](https://sap.github.io/ui5-tooling/stable/pages/CLI/#common-options) command parameter might be inconvenient and even impossible.
+
+#### Resolution
+
+Replace UI5 Tooling's `--log-level` option with the `UI5_LOG_LVL` environment variable.
+
+Example:
+
+`UI5_LOG_LVL=silly ui5 build`
```

Comment:
Maybe we can provide an *additional* example for the karma use case? I.e. `UI5_LOG_LVL=verbose npm run karma`

Reply:
Thanks!